client/wfprev-war/src/main/angular/src/app/components/list-panel/projects-list/projects-list.component.html

```diff
@@ -0,0 +1,127 @@
+<div>
+  <div class="list-header">
+    <!-- Sort Dropdown -->
+    <div class="sort-dropdown">
+      <label>Sort:</label>
+      <select id="sort" (change)="onSortChange($event)">
+        <option class="default-option" value="">select...</option>
+        <option *ngFor="let option of sortOptions" [value]="option.value">{{ option.label }}</option>
+      </select>
+    </div>
+
+    <!-- Sync Toggle -->
+    <div class="sync-toggle">
+      <div class="label-container">
+        <label>Sync with Map</label>
+      </div>
+      <div class="toggle-container">
+        <mat-slide-toggle color="primary" (change)="onSyncMapToggleChange($event)">
+          <span>Off/On</span>
+        </mat-slide-toggle>
+      </div>
+    </div>
+
+    <!-- Result Count -->
+    <div class="result-count">
+      <span>{{ resultCount }} Results</span>
+    </div>
+  </div>
+
+  <div class="list-contents">
+    <mat-accordion>
+      
+      <mat-expansion-panel *ngFor="let item of projectList" #panel="matExpansionPanel">
```

Comment:
instead of naming each thing an "item", can we call them "project" for more clarity for the reader?

Reply:
done